controllers/webspherelibertyapplication_controller.go

```diff
@@ -530,6 +530,14 @@ func (r *ReconcileWebSphereLiberty) Reconcile(ctx context.Context, request ctrl.
 		reqLogger.V(1).Info(fmt.Sprintf("%s is not supported", prometheusv1.SchemeGroupVersion.String()))
 	}
 
+	instance.Status.Versions.Reconciled = lutils.OperandVersion
```

Comment:
Just setting this line should be enough. The `ManageSuccess` call should write out the status. 

There is another ManageSuccess call above (for Knative scenario). The reconciled version should be added prior to it as well.